src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Receivers.kt

```diff
@@ -161,22 +159,24 @@ class MarkdownTaskReceiver(
             return false
         }
 
+        val convertedMarkdown = File(commandArguments["outputFileName"])
+        val response = compileMarkdown(request, convertedMarkdown)
         val taskId = request.id
 
         val onPublishFailureCallback = {
             LOGGER.info("Publish $taskId failed with code ${StatusCode.FAILURE}")
         }
 
-        val data = getResultData(taskId, ByteString.copyFrom(MOCK_PDF_BYTES),
-                MOCK_FILE_NAME, StatusCode.SUCCESS.ordinal)
+        val data = getResultData(taskId, response.pdfFile,
+                Files.getNameWithoutExtension(request.mainFileName) + ".pdf", response.status.ordinal)
```

Comment:
If you sure that ordinal numbers of enum members from `CompileResponse.Status` are exactly the same as one would expect to find in `CompilerFleetResponse.statusCode` then you may want to replace status code with that enum, either by copying the enumeration to new proto or by referencing it somehow

Reply:
Is it appropriate to use exactly the same `CompileResponse.Status` in `CompilerFleetResult`?
I mean, otherwise I need to copy `CompileResponse.Status` definition to `CompilerFleetResult` and setting field(via `response.status.ordinal`) would be the same.

Reply:
Like this?